Packs/Cisco-umbrella/Integrations/CiscoUmbrellaEnforcement/CiscoUmbrellaEnforcement.py

```diff
@@ -0,0 +1,197 @@
+    device_id = args.get('device_id')
+    dst_domain = args.get('destination_domain')
+    dst_url = args.get('destination_url')
+    device_version = args.get('device_version')
+
+    new_event = {
+        "alertTime": alert_time,
+        "deviceId": device_id,
+        "deviceVersion": device_version,
+        "dstDomain": dst_domain,
+        "dstUrl": dst_url,
+        "eventTime": alert_time,
+        "protocolVersion": "1.0a",
+        "providerName": "Security Platform"
+    }
+
+    response: dict = client.add_event_to_domain(new_event)
+    if id := str(response.get('id')):
+        action_result = f"New event was added successfully, The id is {id}."
+    else:
+        action_result = "New event's addition failed."
+    return action_result
+
+
+def domain_delete_command(client: Client, args: dict) -> CommandResults:
+    """
+    :param client: Cisco Umbrella Client for the api request.
+    :param args: args from the user for the command.
+    :returns (str) confirmation or error regarding deleting a domain.
+    """
+    response = {}
+    domain_name = args.get('name', '')
+    domain_id = args.get('id', '')
+    if not domain_name and not domain_id:
+        raise DemistoException(
+            'Both domain name and domain id do not exist, Please supply one of them in order to set the domain to '
+            'delete command')
+    try:
+        response = client.delete_domains(domain_id=domain_id, domain_name=domain_name)
+    except DemistoException as e:
+        if e.res.get('statusCode') == 404:
+            return CommandResults(
+                readable_output='The domain was not found in the list, Please insert an existing domain name or id.'
+            )
+    if domain_name:
+        old_context = demisto.dt(demisto.context(), f'CiscoUmbrellaEnforcement.Domains(val.name == "{domain_name}")')
+    else:
+        old_context = demisto.dt(demisto.context(), f'CiscoUmbrellaEnforcement.Domains(val.id == "{domain_id}")')
+
+    demisto.info(f'old context {str(old_context)} and domain name is {domain_name} and domain id is {domain_id}')
+    if old_context:
+        if isinstance(old_context, list):
+            old_context = old_context[0]
+        old_context['IsDeleted'] = True
+    if response and int(response.status_code) == 204:
+        message = f"{domain_name if domain_name else domain_id} domain was removed from blacklist"
+    else:
+        message = f"{domain_name if domain_name else domain_id} domain not in the blacklist or Error"
+    return CommandResults(
+        readable_output=message,
+        outputs_prefix='CiscoUmbrellaEnforcement.Domains',
+        outputs_key_field='id',
+        outputs=old_context
+    )
+
+
+def test_module(client: Client) -> str:
+    """
+    :param client: Cisco Umbrella Client for the api request.
+    :return: 'ok' if there is a connection with the api and exception otherwise.
+    """
+    client.get_domains_list(limit='1', page='1')
```

Comment:
with accordance to the prev int suggestion

```suggestion
    client.get_domains_list(limit=1, page='1')
```